USE-CASES.md

```diff
@@ -180,6 +183,77 @@ Such questions are answerable if the defender has the requisite “capabilities,
+
+##### Use cases
+Account Creation: Seeing an identity being used with a device it has never been seen with before (or that's very different from the devices it has been seen with before) can indicate identity theft where a fraudster uses a known identity to open new accounts.
+Account Takeover: An account that is taken over will be accessed from a new device, i.e. a device it hasn’t been used with before. This can be treated as an additional risk compared to devices that have been regularly seen with said account. The lifespan during which a device can be recognized as ‘seen again’ is very relevant here, as shorter windows will make every account access look as if it came from a never-before-seen device.
+1st Party Fraud: A person committing 1st party fraud and then claiming to be a victim of identity theft can in some cases be detected by recognizing that their own device was used to commit said fraud.
+
+####  Recognize whether the same device is seen again in the context of multiple identities
+Similar to the above, but recognizing the usage of the same device with different identities
+
+##### Use cases
+Account Creation: Fraudsters often steal multiple identities. Seeing many different identities’ data coming from the same device is a high risk signal, since legitimate users mostly use a single identity with their device.
+Account Takeover: A similar point to the above holds for account takeover when e.g. a fraudster is using a list of leaked usernames and passwords to takeover multiple victims' accounts. Recognizing that these logins into multiple accounts are happening from the same device can be used to indicate high risk.
+
+#### Retrieve a device’s IP address
+Knowledge of the IP address offers many benefits in the fight against identity fraud:
+
+VPN/TOR detection: Many cases of fraud are committed from devices on VPNs. This is therefore a valuable risk signal.
+Relatively unique identifier: Similar to the use cases above. For example: 
+Account Creation: seeing unusual numbers of account creations from the same IP or IP range indicates risk; 
+Account Takeover: seeing an identity with an IP address that it has never been associated with. These are ways in which knowledge of the IP address can contribute to risk insights.
+IP block nature: If certain IPs within a block have been associated with fraud, the overarching block can be preemptively labeled as more risky.
+Approximate location information: IP addresses are associated with geolocation information. During account creation if an IPs geolocation is far removed from the submitted address this indicates higher risk of fraud.
+
+#### Know the geographic location of the device
+This refers to geolocation information through GPS / Wi-Fi triangulation, as this offers additional confidence over only IP-based geolocation, particularly when proxies are being used.
+
+#### Know that the geographic location of a device is being manipulated
```

Comment:
Hi Philip, 

I believe this was submitted by someone else, but from an identity fraud perspective I believe there is a need to keep these two separate. Location spoofing is a very popular technique that fraudsters use. Even when assuming that geolocation x provided by the "Know the geographic location of the device" capability is the correct location and not the spoofed location, there is still relevant information in knowing the spoofed location or at a minimum know that spoofing happened. 
"Know that the geographic location of a device is being manipulated" causes a distinction between e.g. login attempts by a person who is traveling to geolocation x (and not spoofing) versus a fraudster who is present at geolocation x but is also spoofing.

Reply:
I imagined this as you getting an attested location X that you can compare with the system-provided location Y.
If X != Y, you know that the location is being manipulated. If our attestation mechanism _is_ being manipulated and X == Y, the attestation mechanism would presumably not be able to detect this manipulation (or else it would resist the manipulation).

We can keep these separate, but I wanted to make sure my reasoning here isn't missing something.

Reply:
(I'm ok resolving this comment, we can revisit the nuances of how deception is prevented in location once this topic becomes more focal)